### PR TITLE
Release docs for the second release of 2019 June

### PR DESCRIPTION
**Pull request by gallayl**

### CODE REVIEW

File: _updates/2019-06-24-june_2-2019.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+### @sensenet/client-utils
+ - fixed deepMerge implementation for falsy but defined values
+
+### @sensenet/client-core
+ - added GET method parameter parsing from the action body
+ - optional **requestInit** parameter on **repository.security.getParentGroups()** for request cancellation
+ - new action for **repository.getWopiData()**
+ 
+### @sensenet/authentication-google
+ - fixed sanbox attribute's value
```
Comment (herflis) [inline _updates/2019-06-24-june_2-2019.md]:
```suggestion
 - fixed sandbox attribute's value
```

File: _updates/2019-06-24-june_2-2019.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ 
+### @sensenet/controls-react
+ - Fixes in ColorPicker control
+ - Fixes in Allowed Child Types control
+
+### sn-dms-demo@
```
Comment (herflis) [inline _updates/2019-06-24-june_2-2019.md]:
```suggestion
### sn-dms-demo
```

File: _updates/2019-06-24-june_2-2019.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+ - Office Online integration
+ - changed the default app service URL to our ``devservice`` repository
+ - varioius search fixes
+
+### @sensenet/sn-app@
```
Comment (herflis) [inline _updates/2019-06-24-june_2-2019.md]:
```suggestion
### @sensenet/sn-app
```

File: _updates/2019-06-24-june_2-2019.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ - fixed sanbox attribute's value
+ 
+### @sensenet/controls-react
+ - Fixes in ColorPicker control
+ - Fixes in Allowed Child Types control
+
+### sn-dms-demo@
+ - Office Online integration
+ - changed the default app service URL to our ``devservice`` repository
+ - varioius search fixes
```
Comment (zoltanbedi) [inline _updates/2019-06-24-june_2-2019.md]:
```suggestion
 - various search fixes
```

**Review by zoltanbedi — APPROVED** [review]
📄

File: _updates/2019-06-24-june_2-2019.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+title: June 2019 / 2
+---
+
+Another 4 weeks went by, here are the changes we made 👇🏻
+
+## Mononorepo changes
```
Comment (tusmester) [inline _updates/2019-06-24-june_2-2019.md]:
```suggestion
## Monorepo changes
```

**Review by tusmester — CHANGES_REQUESTED** [review]

**Pull request merged**